src/praisonai/praisonai/gateway/server.py

```
@@ -2056,6 +2090,64 @@ def health(self) -> Dict[str, Any]:
+
+        A channel that is paused/stopped is treated as not-blocking readiness;
+        only FAILED channels (unrecoverable or actively erroring) are reported.
+        """
+        unhealthy: List[Tuple[str, Any]] = []
+        try:
+            from .supervisor import ChannelState
+            for name, status in self._channel_supervisor.get_all_status().items():
+                if status.state == ChannelState.FAILED:
+                    unhealthy.append((name, status))
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)